### ISSUE TITLE
Flatseal 2.3.0 failed to run

### ISSUE THREAD

**Issue by ghost** [tchx84/Flatseal#726]
Error message:

MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:763: FINISHME: support YUV colorspace with DRM format modifiers
MESA-INTEL: warning: ../src/intel/vulkan/anv_formats.c:794: FINISHME: support more multi-planar formats with DRM modifiers
Gdk-Message: 04:10:45.403: Error 71 (Protocol error) dispatching to Wayland display.

**Comment by rusty-snake** [comment]
Duplicate of #725

**Comment by rusty-snake** [comment]
TL;DR: Wait for runtime update or workaround with `GSK_RENDERER=gl`.

**Comment by tchx84** [comment]
@rusty-snake thanks, and let's keep this open for a while.. I can see more of these reports coming.

**Comment by tchx84** [comment]
See the GTK issue [here](https://gitlab.gnome.org/GNOME/gtk/-/issues/7047).

TLDR: update the gnome runtime e.g., `flatpak update org.gnome.Platform` (or else use a different renderer, e.g., `GSK_RENDERER=ngl flatpak run com.github.tchx84.Flatseal`).

**Comment by izmyname** [comment]
Yup, it's not just flatseal - everything gtk is borked.

**Comment by Angxddeep** [comment]
@tchx84 So I looked into it, it's a vulkan issue with nvidia proprietary driver on hybrid systems with intel/nvidia or amd/nvidia setup, and the Gnome guys were like 'we can't do anything if the nvidia driver has a bug' so i don't see it resolving for a while.

https://gitlab.freedesktop.org/mesa/mesa/-/issues/11723

**Comment by ghost** [comment]
I have the same problem, also in other apps like bottles. For some reason it was solved by blocking wayland in overrides.

**Comment by kRHYME7** [comment]
> For some reason it was solved by blocking wayland in overrides.

or this 
```bash 
 GSK_RENDERER=ngl flatpak run com.github.tchx84.Flatseal 
```

or if you can access flat seal add it in here
![image](https://github.com/user-attachments/assets/ba289279-45ff-40c4-8ebc-841b3bf23522)

**Comment by Angxddeep** [comment]
@tchx84 this got fixed with the release of   Nvidia 565 drivers. SO you can close this issue now.

**Comment by synthfi** [comment]
automatic update nvidia driver update from 560 to 570 fixed this for me on ubuntu 👍 I have a hybrid amd + nvidia setup

**Comment by tchx84** [comment]
@Angxddeep @synthfi thanks!

**Comment by h-town** [comment]
> [@tchx84](https://github.com/tchx84) this got fixed with the release of Nvidia 565 drivers. SO you can close this issue now.

When asked for comment mere weeks after upgrading to the latest and greatest Debian release "Trixie", itself delivered with NVIDIA 550 series drivers not due for a step-change upgrade until 2027, one long time Debian user replied, "Look man, I'll gladly forgo any semblance of modern desktop computing in favor of a 2 year moratorium on features. With that upgrade behind me, now I can get back to dividing my time between proselytizing the evils of Frankendebian to strangers on the Internet  and quietly spending days of cumulative man hours over the next 2 years in pursuit of Flatpak/Snap/AppImage/gcc/RandomInternetBinary-fueled feature parity."

Only 22 months 'til Forky baby!! 🎉😔
